Signal/src/ViewControllers/DebugUI/DebugUIMessagesAssetLoader.h

```diff
@@ -0,0 +1,35 @@
+//
+//  Copyright (c) 2018 Open Whisper Systems. All rights reserved.
+//
+
+#import "DebugUIMessagesUtils.h"
+
+NS_ASSUME_NONNULL_BEGIN
+
+@interface DebugUIMessagesAssetLoader : NSObject
```

Comment:
These debug actions often require assets for attachments, either downloaded or generated on the fly.  This class handles that and:

* Caches on disk to ensure we only download/create these assets once per install.